forms/chapter-organizer-form.md

```diff
@@ -0,0 +1,70 @@
+# PyLadies Chapter Organizer Registration Form
+
+Please fill in this form if you are:
+
+- Currently a PyLadies chapter organizer
+- Was a chapter organizer and wants to step down
+- A new organizer for an existing chapter.
+
+Please share this form with your co-organizers, as well as please remember
+to fill this form again with any changes in your chapter (i.e. someone
+new is joining the team or someone is stepping down).
+
+We may request you to fill this form annually, to keep our records up to date.
+
+## How will the data here be used?
+
+PyLadies [Admin-and-logistics team members](https://github.com/pyladies/project-admin-logisitics#team)
+will refer to this data if we need to get in touch
+with local chapter organizers, for example for verifying the chapter's status
+or for verifying password reset request.
+
+Other PyLadies members and organizers may want to contact a chapter directly
+for various purposes, for example: sharing conference discount codes,
+sharing job opportunities, request speakers from your community, et.
+
+## Who has access to this form?
+
+The information collected here will be made available to all PyLadies chapter
+organizers, as a Google Spreadsheet. The Google Spreadsheet can be viewed
+by anyone who logged in using the `{chapter}@pyladies.com` account.
+
+If you are a chapter organizer and would like access to this data,
+you can go to "(url tbd)" and log in using your chapter's `{chapter@pyladies.com}`
+email address.
+
+## Questions and concerns?
+
+Please bring this up in: <https://github.com/pyladies/info> and open an issue.
+
+Est. 2020
+
+# Form Questions
+
+- (Personal) Email address: (textbox)
+
+  Your own personal email address, which we can use to contact you
+  
+- PyLadies.com Email address: (textbox)
+
+  Your chapter's pyladies.com email address. Example: `vancouver@pyladies.com`
+
+- Your Name: (textbox)
+
+- Your pronouns: (options)
```

Comment:
Recommend:

- making this optional (for people who chose not to disclose their pronouns)
- either adding in "None of the Above" or better yet, a "Add yours here" (data will be more mixed, but representation is most important)

Reply:
I'm curious about why sharing pronouns would be optional? I have yet to come across that.

Reply:
I've added the "Other" option.